src/app/tests/TestSceneTable.cpp

```diff
@@ -119,6 +121,24 @@ class TestSceneHandler : public scenes::DefaultSceneHandlerImpl
     TestSceneHandler() = default;
     ~TestSceneHandler() override {}
 
+    // Fills in cluster buffer and adjusts its size to lower than the maximum number of cluster per scenes
+    virtual void GetSupportedClusters(EndpointId endpoint, Span<ClusterId> & clusterBuffer) override
+    {
+        ClusterId * buffer = clusterBuffer.data();
+        if (endpoint == TEST_ENDPOINT1)
+        {
+            buffer[0] = ON_OFF_CID;
+            buffer[1] = LV_CTR_CID;
```

Comment:
Must ensure than `clusterBuffer.size()` is big enough before copying.

Reply:
Suggestion was applied in latest commit